Woof/WoofTests/ViewModel/UserRoleViewModelTests/UserRoleViewModelTests.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+
+final class UserRoleViewModelTests: XCTestCase {
+    func testUserRoleManagerExistInAPI() {
+        _ = UserRoleViewModel()
```

Comment:
Please update the name of the test methods, they don't match after renaming.

Reply:
Done.